Blog post recap and highlights Prague 22 (#1809)
writethedocs/www: docs/conf/prague/2022/news/thank-you-recap.rst

:template: {{year}}/generic.html
:banner: _static/conf/images/headers/venue.png

.. post:: Sept 23, 2022
   :tags: recap, thanks, {{shortcode}}-{{year}}

Conference highlights and thanks for attending!
===============================================

Hey everyone,

Our team is now emerging from the post-conference recovery period, and we wanted to send out a quick message to thank everyone for helping to make our conference such a success. 

> We appreciate everyone who makes the conference possible: speakers, sponsors, organizers, and attendees alike!

Reviewer: Could add volunteers here?

Author: Welcome to add a suggestion directly ;)

Author: Ok I'll add it, thanks for the notes!

> We appreciate everyone who makes the conference possible: speakers, sponsors, organizers, and attendees alike!

Author: ```suggestion
We appreciate everyone who makes the conference possible: speakers, sponsors, organizers, volunteers, and attendees alike!
```

Conference recap
----------------

At **over 200 attendees** this conference was a welcoming and friendly event.
We had the usual mix of talks on the main stage along with Q&A, lightning talks, unconference sessions, writing sprints, social event, and job fair.
We thought everything went pretty smoothly this year; we're getting pretty good at virtual events!

Talk videos
-----------

`Videos of the talks`_ are now available on YouTube. This includes all conference talks and Q&A, as well as the lightning talks.
You can also subscribe to our `YouTube channel`_ so you don’t miss any future content.

> Thanks again to all our presenters, moderator, and folks who asked wonderful questions!

Author: ```suggestion
Thanks again to all our presenters, moderators, and folks who asked wonderful questions!
```

Huge thanks also to `Bart at Work`_ for handling the A/V production on our event, it was flawless once again this year.

.. _Videos of the talks: https://youtube.com/playlist?list=PLZAeFn6dfHpm1PRgp84X5jh9Jca_KTJSF
.. _Bart at Work: https://www.bartatwork.com/
.. _YouTube channel: https://www.youtube.com/writethedocs

Sketchnotes
-----------

We’re excited to share the `full resolution sketchnotes`_ from the conference on Flickr. You are welcome to use them for non-commercial purposes, like including in your write up from the conference :)

Thanks to `Linette Voller`_ for doing them again this year!

> .. _full resolution sketchnotes: https://www.flickr.com/photos/writethedocs/albums/72177720299222782

Author: ```suggestion
.. _full resolution sketchnotes: https://flic.kr/ps/2RZ8SH
```

.. _Linette Voller: https://twitter.com/mslanei

> Come see us at our Australia conference

Author: ```suggestion
Shirts still on sale
--------------------

If you want a souvenir from this year's conference, our `Write the Docs Prague 2022 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is still open until September 30th! 
We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches, and shipping is available from the USA or Europe.

Come see us at our Australia conference
```

----------------------------------------------------------

Don’t forget our `Australia conference`_ on **Dec 8-9, 2022**. The talks will be posted soon!

> We’re also looking for more sponsors to make our other events just as magical as this year’s Prague event. You can see all our `sponsorship options`_ on our website.

Reviewer 2: Thanks for the shout out for sponsors for the Australian conference Mikey

.. _Australia conference: https://www.writethedocs.org/conf/australia/2022/
.. _sponsorship options: https://www.writethedocs.org/sponsorship/

Write the Docs Slack
--------------------

Our Slack network has really grown more this last year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y’all continue the conversations throughout the year. If you haven’t joined, now is the time! 
Check out the `Write the Docs Slack`_.

.. _Write the Docs Slack: http://www.writethedocs.org/slack/

Write the Docs Newsletter
-------------------------

Our `monthly newsletter`_ is the other place to keep track of everything that is happening with the community. 
Subscribe to make sure you hear about all the conferences, meetups, job postings, and summaries of relevant conversations in our Slack.

.. _monthly newsletter: http://writethedocs.org/newsletter/

Local Meetups
-------------

> We hope you continue to `build the meetup community`_ and keep in touch with other documentarians near your hometown. 

Reviewer: ```suggestion
We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
```

Huge thanks also to `Bart at Work`_ for handling the A/V production on our event, it was flawless once again this year.

.. _Videos of the talks: https://youtube.com/playlist?list=PLZAeFn6dfHpm1PRgp84X5jh9Jca_KTJSF
.. _Bart at Work: https://www.bartatwork.com/
.. _YouTube channel: https://www.youtube.com/writethedocs

Sketchnotes
-----------

We’re excited to share the `full resolution sketchnotes`_ from the conference on Flickr. You are welcome to use them for non-commercial purposes, like including in your write up from the conference :)

Thanks to `Linette Voller`_ for doing them again this year!

.. _full resolution sketchnotes: https://flic.kr/ps/2RZ8SH
.. _Linette Voller: https://twitter.com/mslanei

Shirts still on sale
--------------------

If you want a souvenir from this year's conference, our `Write the Docs Prague 2022 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is still open until September 30th! 
We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches, and shipping is available from the USA or Europe.

Come see us at our Australia conference

> ----------------------------------------------------------

Author: ```suggestion
---------------------------------------
```

--------------------

Our Slack network has really grown more this last year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y’all continue the conversations throughout the year. If you haven’t joined, now is the time! 
Check out the `Write the Docs Slack`_.

.. _Write the Docs Slack: http://www.writethedocs.org/slack/

Write the Docs Newsletter
-------------------------

Our `monthly newsletter`_ is the other place to keep track of everything that is happening with the community. 
Subscribe to make sure you hear about all the conferences, meetups, job postings, and summaries of relevant conversations in our Slack.

.. _monthly newsletter: http://writethedocs.org/newsletter/

Local Meetups
-------------

We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!

New this year are our `Quorum Meetups`_ which are virtual and targeting large geographic areas. There’s already one covering most everyone in the US or in a similar timezone.

> .. _build the meetup community: http://www.writethedocs.org/meetups/

Author: ```suggestion
.. _build your local or online communities: http://www.writethedocs.org/meetups/
```
